test/openshift/e2e/sequential/1-078_validate_default_argocd_consoleLink/03-check-env.yaml

```diff
@@ -2,7 +2,16 @@ apiVersion: kuttl.dev/v1beta1
 kind: TestStep
 commands:
 - script: |
-    if [ -z $CI ]; then
+    if ! [ -z $NON_OLM ]; then
+      label_value='"true"'
+      if ! [[ "${label_value}" == '"true"' ]]; then
+        echo "Label value: $label_value"
+        echo "DISABLE_DEFAULT_ARGOCD_CONSOLELINK not set"
```

Comment:
This checks do not make sense. Shouldn't we retrieve the value first here?

Reply:
corrected